.travis.yml

```diff
@@ -1,14 +1,23 @@
 language: python
 python:
-- 3.8
+- '3.8'
 env:
-- TOXENV=django22
 - TOXENV=quality
 - TOXENV=pii-annotations
+- TOXENV=django22
+- TOXENV=django30
+- TOXENV=django31
+- TOXENV=django32
 matrix:
   include:
-  - python: 3.8
+  - python: '3.8'
```

Comment:
we don't need this `include` section if we're running all tox envs against python 3.8